plugins/SolidView/xray_composite.shader

```diff
@@ -39,6 +40,13 @@ fragment =
 
     float kernel[9];
 
+    float hash12(vec2 p)
```

Comment:
Waar gebruik je deze functie(s)?

Reply:
In een oude commit. Ik heb ze verwijderd.